api/plugins/module_utils/api_client.py

```diff
@@ -563,6 +563,35 @@ def user_remove_group(self, email, group_name):
 
         return self.make_api_call(self.__prepare_graphql_query(query) % (email, group_name))
 
+    def deploy_target_config_get(self, project_name):
+       query = {
+           'query': """query projectInfo($name: String!) {
+               projectByName(name: $name) {
+                   id
+                   deployTargetConfigs {
+                       id
+                       weight
+                       branches
+                       pullrequests
+                       deployTarget {
+                           name
+                           id
+                       }
+                       project{
+                           name
+                       }
```

Comment:
I don't think we need this, it would add to the query time.
```suggestion
```

Reply:
Tested and this works 👍🏼